lib/fish/drivers.go

```diff
@@ -22,7 +22,8 @@ import (
 	// Load all the drivers
 	_ "github.com/adobe/aquarium-fish/lib/drivers/aws"
 	_ "github.com/adobe/aquarium-fish/lib/drivers/docker"
-	_ "github.com/adobe/aquarium-fish/lib/drivers/native"
+	_ "github.com/adobe/aquarium-fish/lib/drivers/native/mac"
```

Comment:
Still will need to be in the same folder, otherwise native driver will be split and we need to have it together - just different implementations for the platforms.